src/components/explore-projects/ExplorePojects.js

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import ImageList from "@mui/material/ImageList";
+import ImageListItem from "@mui/material/ImageListItem";
+import ProjectImageDisplay from "./project-image-display/ProjectImageDisplay";
+import projectData from "../../utils/assets";
+import exploreProjectsStyles from "./exploreProjectsStyles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import exploreProjectsUtils from "./exploreProjectsUtils";
+const projectImages = projectData;
+
+exploreProjectsUtils.handleProjectImagesGrid();
```

Comment:
what is this doing here? can we avoid doing this?

Reply:
Yes we can, nice catch.